tests/suite/custom_resources_utils.py

```diff
@@ -214,10 +214,9 @@ def delete_policy(custom_objects: CustomObjectsApi, name, namespace) -> None:
     :return:
     """
     print(f"Delete a Policy: {name}")
-    delete_options = client.V1DeleteOptions()
 
     custom_objects.delete_namespaced_custom_object(
-        "k8s.nginx.org", "v1", namespace, "policies", name, delete_options
+        "k8s.nginx.org", "v1", namespace, "policies", name
```

Comment:
are delete_options not needed unlike https://github.com/nginxinc/kubernetes-ingress/pull/1522/files#diff-b419599ec413dec04c511421059de8ee7b224c490b5785df0dcf2edb50394217R674-R679?